timeutil/schedule.go

```diff
@@ -708,3 +708,47 @@ func parseEventSet(s string) (*Schedule, error) {
 
 	return &schedule, nil
 }
+
+// Includes checks whether given time t falls inside the time range covered by
+// the schedule. A single time schedule eg. '10:00' is treated as spanning the
+// time <10:00, 10:01)
+func (sched *Schedule) Includes(t time.Time) bool {
+	if len(sched.WeekSpans) > 0 {
+		var weekMatch bool
+		for _, week := range sched.WeekSpans {
+			if week.Match(t) {
+				weekMatch = true
+				break
+			}
+		}
+		if !weekMatch {
+			return false
+		}
+	}
+
+	for _, tspan := range sched.flattenedClockSpans() {
+		window := tspan.Window(t)
+		if window.End.Equal(window.Start) {
+			// schedule granularity is a minute, a schedule '10:00'
+			// in fact is: <10:00, 10:01)
```

Comment:
Curious notation.. where does it come from?

The typical math convention I remember would be _[10:00, 10:01)_

Reply:
This may be the old Polish/Russian/Czech textbook notation, eg. `〈1, 2 〉`meaning closed interval from 1 to 2, i.e. `[1,2]` in the ISO/standard (?) notation. `(1,2)` is has the same meaning in both notations (although IIRC French notation was more like `] 1,2 [`). I'll fix it to use the standard one.